guides/creating-a-custom-form-field-control.md

```diff
@@ -227,23 +227,30 @@ For additional information about `ControlValueAccessor` see the [API docs](https
 This property indicates whether the form field control should be considered to be in a
 focused state. When it is in a focused state, the form field is displayed with a solid color
 underline. For the purposes of our component, we want to consider it focused if any of the part
-inputs are focused. We can use the `FocusMonitor` from `@angular/cdk` to easily check this. We also
-need to remember to emit on the `stateChanges` stream so change detection can happen.
+inputs are focused. We can use the `focusin` and `focusout` events to easily check this. We also
+need to remember to emit on the `stateChanges` when the focused stated changes stream so change
+detection can happen.
+
+In addition to updating the focused state, we use the `focusin` and `focusout` methods to update the
+internal touched state of our component, which we'll use to determine the error state.
 
 ```ts
 focused = false;
 
-constructor(fb: FormBuilder, private fm: FocusMonitor, private elRef: ElementRef<HTMLElement>) {
-  ...
-  fm.monitor(elRef, true).subscribe(origin => {
-    this.focused = !!origin;
+onFocusIn(event: FocusEvent) {
```

Comment:
optional: my preference would be to name these methods something that reflects what they do, but I don't feel super strongly since it's an example